src/instructlab/model/chat.py

```diff
@@ -789,18 +797,41 @@ def chat_cli(
     sys_prompt = CONTEXTS.get(context, "default")(get_model_arch(pathlib.Path(model)))
     loaded["messages"] = [{"role": "system", "content": sys_prompt}]
 
+    retriever: DocumentStoreRetriever | None = None
+
     # Instantiate retriever if RAG is enabled
     if rag_enabled:
         logger.debug("RAG enabled for chat; initializing retriever")
-        retriever: DocumentStoreRetriever | None = create_document_retriever(
-            document_store_uri=document_store_uri,
-            document_store_collection_name=collection_name,
-            top_k=top_k,
-            embedding_model_path=embedding_model_path,
-        )
+        if not os.path.exists(embedding_model_path) or not os.access(
+            embedding_model_path, os.R_OK
+        ):
+            raise ChatFailedToLoadRetrievalModelException(
```

Comment:
Shouldn't we also print a message to console like the following?
(I'm not 100% sure about this because it's not consistently adopted in the existing code)
```py
            self.console.print(
                display_wrapper(...), style="bold red"
            )
```